docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,114 @@
+
+.. _WCAG 2.2: https://www.w3.org/TR/WCAG22/
+.. _WCAG 3.0: https://www.w3.org/TR/wcag-3.0/
+.. _ATAG 2.0: https://www.w3.org/TR/ATAG20/
+.. _The A11Y Project: https://www.a11yproject.com/
+
+.. _accessibility-support-targets:
+
+Accessibility support targets
+=============================
+
+Beyond standards, we want to make sure Django actually works for people using
+a wide range of input devices and assistive technologies.
+The best way to do this effectively is to take accessibility considerations
+into account as part of designing features. If in doubt, consult
+with users who rely on assistive technologies or with accessibility experts.
+
+.. _accessibility-testing-baseline:
+
+Accessibility testing baseline
+------------------------------
+
+ Always test user interface changes with:
+
+* Keyboard-only navigation
+* Automated checks in the `Accessibility Insights`_ browser extension,
+  or an equivalent tool with the `Axe`_ checker.
+
+Where appropriate, also test with:
+
+* Touch-only navigation
+* 400% browser zoom
+* forced-colors mode DevTools emulation
+
+.. note::
+    How much should you test? Devise the UI with accessibility in mind,
+    and the testing will only be needed as a final check. For more complex
+    interfaces, confer with other contributors to decide on testing targets.
+
+.. _assistive-technologies:
+
+Assistive technologies
+----------------------
+
+Where more extensive testing is appropriate,
+here are free assistive technologies we recommend testing with:
+
+* `NVDA`_ on Windows with Firefox ESR
+* `VoiceOver`_ on macOS with Safari
+* `Narrator`_ on Windows with Microsoft Edge
+* Windows Magnifier and macOS Zoom
+* Windows Speech Recognition and macOS Voice Control
+* VoiceOver on iOS, or TalkBack on Android
+* `Contrast themes`_ on Windows
+
+Where expert help is available, also consider testing with:
+
+* `JAWS`_ on Windows
+* `Dragon`_ on Windows
+
+.. _Accessibility Insights: https://accessibilityinsights.io/
+.. _Axe: https://github.com/dequelabs/axe-core
+.. _NVDA: https://www.nvaccess.org/
+.. _Narrator: https://support.microsoft.com/en-us/windows/complete-guide-to-narrator-e4397a0d-ef4f-b386-d8ae-c172f109bdb1
+.. _VoiceOver: https://support.apple.com/en-gb/guide/voiceover-guide/welcome/web
+.. _TalBack: https://support.google.com/accessibility/android/answer/6283677
+.. _Contrast themes: https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696
+.. _JAWS: https://www.freedomscientific.com/products/software/jaws/
+.. _Dragon: https://www.nuance.com/en-gb/dragon.html
+
+.. _accessibility-known-issues-how-to-help:
+
+Known issues and how to help
+============================
+
+Parts of Django do not meet our accessibility targets at the moment.
+We actively work on fixing issues both as part of ongoing maintenance,
+and bigger overhauls. To learn about known issues, and get involved, see:
```

Comment:
I thought it was important to acknowledge that Django is quite far from meeting all the goals stated above (it’ll be plenty obvious to people going through the Django admin with our stated standards and testing targets in particular) – but that’s something we’re working on (and _you_ can get involved 😉 ).

Reply:
Maybe a link to a board of open issues would be cool, such that possible maintaners can _jump in_

Reply:
There are links to open issues in items 2 and 3 in the list below, is that different from what you’re expecting?